code/modules/clothing/suits/armor.dm

```diff
@@ -76,6 +76,11 @@
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS|LEGS
 	price_tag = 350
 
+/obj/item/clothing/suit/armor/vest/warden/ironhammer
+	name = "Warden's overcoat"
```

Comment:
Half of items having jobs designations are capitalized, the other half is not.
Let's just follow the parent item (capitalized) for this item